deploy.bat

```diff
@@ -0,0 +1,2 @@
+yarn build
+cf push [YOUR_APP_NAME]
```

Comment:
I understand your point (i also like the cf target in our makefile in internal services :) ). But i prefere to not include the cf target at the moment, because one command above in this readme, the user should define the org and space with cf target. Maybe include this later with the bootstrap/setup script idea.